<!-- use this launch anytime the one controller is needed -->
<launch>

<rosparam command="load" file="$(find stewart_platform)/config/x360.yaml" />

<node respawn="true" pkg="joy" type="joy_node" name="joy" >
	<param name="dev" type="string" value="/dev/input/js0" />
	<param name="deadzone" value="0.2" />
</node> 


  <node pkg="stewart_platform" type="stewart_platform_x360.py" name="stewart_platform_x360" output="screen">
  	<remap from="/X360/Joy" to="joy" />
  </node>

</launch>
